lib/resources/r4/communication.yaml

```diff
@@ -134,10 +130,10 @@ fields:
 - name: recipient
   required: 'Yes'
   cardinality: 1..*
-  type: Reference(Practitioner/Group)
+  type: Reference (Practitioner, Group)
   description: The Practitioner or Pool (Group) that the communication is intended to reach
```

Comment:
I would make sure that we want to say pool here. Kind of leaking logic

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/files/4670196dfe7d6acd28005f6e8fe73f0a080d37f3..cbf73631d007bdd6cc4c8f9bdc432cffd7047e54#diff-f08ae1e4c063b9a36a22adaa43fecf21L134-L135